homeassistant/components/vicare/binary_sensor.py

```diff
@@ -44,13 +52,39 @@
         CONF_DEVICE_CLASS: DEVICE_CLASS_POWER,
         CONF_GETTER: lambda api: api.getCompressorActive(),
     },
+    SENSOR_HEATINGROD_OVERALL: {
+        CONF_NAME: "Heating rod overall",
+        CONF_DEVICE_CLASS: DEVICE_CLASS_POWER,
+        CONF_GETTER: lambda api: api.getHeatingRodStatusOverall(),
```

Comment:
Does this provide a boolean (binary) or an actual value? The same for the other following `SENSOR_HEATINGROD_...`.

Reply:
Yes those are all boolean values.

Reply:
Ah ok, I'm guessing no `CONF_UNIT_OF_MEASUREMENT` as it will probably default to On/Off.

Reply:
Well binary sensors don't have a unit of measurement anyways..